frontend/javascripts/libs/utils.ts

```diff
@@ -900,12 +912,18 @@ export function castForArrayType(uncastNumber: number, data: TypedArray): number
   return data instanceof BigUint64Array ? BigInt(uncastNumber) : uncastNumber;
 }
 
-export function convertNumberTo64Bit(num: number | null): [Vector4, Vector4] {
+export function convertNumberTo64Bit(num: number | bigint | null): [Vector4, Vector4] {
+  const [bigNumHigh, bigNumLow] = convertNumberTo64BitTuple(num);
+
+  const low = convertDecToBase256(bigNumLow);
+  const high = convertDecToBase256(bigNumHigh);
```

Comment:
The function deconstructs a 32 bit into four 8-bit numbers (therefore, base 256). In glsl, this would correspond to a vec4 whose components are typically addressed with r, g, b and a. that's why the names are that way. However, I would not use the term color in the function signature, because these parts don't have anything to do with the rendered color.
Do you still think the name `convertDecToBase256` should be changed?